server/controller/questions.js

```diff
@@ -28,3 +28,28 @@ exports.post = async (request, response) => {
     response.status(500).send('Server Error');
   }
 };
+
+exports.put = async (request, response) => {
+  try {
+    const { id: loggedUserId } = request;
+    const data = request.body;
+    const { userId, question, questionId: id } = data;
+    if (loggedUserId === userId) {
```

Comment:
@HemaSAli 
What happens if there's no authenticated user in this request, i.e. no token passed at all?  Would `loggedUserId` just be undefined / null or does this project have some other middleware that would return a 401?

I'm trying to determine if someone could maliciously edit / delete question data by making a request with no authentication token *and* no userId in the PUT data (so therefore loggedUserId would equal userId).